homework-submission/week2/src/components/list/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import '../list/index.css';
+
+function List({ id, description, deadline, handleRemoveItem }) {

```

Comment:
For a component called List I'd expect it to render multiple list items, your component only renders one list item, try to find a more fitting name :-)